src/__swaps__/screens/Swap/components/ReviewPanel.tsx

```diff
@@ -352,6 +367,36 @@ export function ReviewPanel() {
             </Inline>
           </Animated.View>
 
+          {/* <Animated.View style={[{ height: REVIEW_SHEET_ROW_HEIGHT, justifyContent: 'center' }]}>
+            <Inline wrap={false} horizontalSpace="10px" alignVertical="center" alignHorizontal="justify">
+              <Inline wrap={false} horizontalSpace="12px">
+                <TextIcon color="labelTertiary" height={9} size="icon 13px" weight="bold" width={16}>
+                  🐵
+                </TextIcon>
+                <Inline wrap={false} horizontalSpace="4px">
+                  <Text color="labelTertiary" weight="semibold" size="15pt">
+                    DEGEN MODE
+                  </Text>
+                  <Bleed space="12px">
+                    <ButtonPressAnimation onPress={openFlashbotsExplainer} scaleTo={0.8}>
+                      <Text
+                        align="center"
+                        color={{ custom: opacity(labelTertiary, 0.24) }}
+                        size="icon 13px"
+                        style={{ padding: 12, top: 0.5 }}
+                        weight="semibold"
+                      >
+                        􀅴
+                      </Text>
+                    </ButtonPressAnimation>
+                  </Bleed>
+                </Inline>
+              </Inline>
+
+              <DegenModeToggle />
+            </Inline>
+          </Animated.View> */}
+
```

Comment:
We'll want to change the explainer on this once we enable it.

Reply:
yes waiting for designs